haskell/marlowe.cabal

```diff
@@ -10,7 +10,7 @@ cabal-version:          >= 1.10
 
 executable marlowe
     default-language:   Haskell2010
-    build-depends:      base >=4.9,
+    build-depends:      base,
```

Comment:
I had to remove the `base` and `template-haskell` conditions, as it couldn't find a suitable aeson version. Not sure if this can cause another issue.